pkg/scheduler/plugins/loadaware/load_aware.go

```diff
@@ -325,13 +280,28 @@ func (p *Plugin) Score(ctx context.Context, state *framework.CycleState, pod *co
 			}
 		}
 	}
+	return estimatedUsed, nil
+}
 
-	allocatable, err := p.estimator.EstimateNode(node)
-	if err != nil {
-		return 0, nil
+func filterNodeUsage(usageThresholds, estimatedUsed map[corev1.ResourceName]int64, allocatable corev1.ResourceList, prodPod bool, filterProfile *usageThresholdsFilterProfile) *framework.Status {
+	for resourceName, value := range usageThresholds {
+		if value == 0 {
+			continue
+		}
+		total := getResourceValue(resourceName, allocatable[resourceName])
+		if total == 0 {
+			continue
+		}
+		usage := int64(math.Round(float64(estimatedUsed[resourceName]) / float64(total) * 100))
+		if usage > value {
```

Comment:
to reduce the cyclomatic complexity and improve the readability, consider this pattern:

```go
if usage <= value {
  continue
}
// ...
```